Xamarin.Forms.Core/DependencyService.cs

```diff
@@ -61,7 +65,10 @@ public static class DependencyService
 			if (!DependencyTypes.Contains(targetType))
 				DependencyTypes.Add(targetType);
 
-			DependencyImplementations[targetType] = new DependencyData { ImplementorType = implementorType };
+			lock (Locker)
+			{
```

Comment:
Remove brackets

Reply:
What's wrong with the braces?

Reply:
@PureWeen 
I can remove it, sure. But this file uses "wrong" code style in other places as well :) 
![image](https://user-images.githubusercontent.com/10124814/47420375-f1602680-d786-11e8-9664-0d2b60f81bb3.png)

Reply:
I was always told to remove them per our coding conventions (I think @rmarinho was poking me about it) but if that's not our convention then we can just leave them

Technically the guideline is arbitrary

> We use Allman style braces, where each brace begins on a new line. A single line statement block can go without braces but the block must be properly indented on its own line and must not be nested in other statement blocks that use braces (See issue 381 for examples). One exception is that a using statement is permitted to be nested within another using statement by starting on the following line at the same indentation level, even if the nested using contains a controlled block.

So I'll just stop harassing people about it